setup.py

```diff
@@ -14,14 +14,14 @@
 
 from setuptools import setup, find_packages
 
-_current_jaxlib_version = '0.3.10'
+_current_jaxlib_version = '0.3.11'
 # The following should be updated with each new jaxlib release.
-_latest_jaxlib_version_on_pypi = '0.3.10'
+_latest_jaxlib_version_on_pypi = '0.3.11'
```

Comment:
I think this one isn't supposed to be updated until after the new release is pushed